src/coreclr/tools/Common/TypeSystem/IL/CoreRTILProvider.cs

```diff
@@ -82,7 +82,7 @@ private MethodIL TryGetIntrinsicMethodIL(MethodDesc method)
                             return GetCanonTypeIntrinsic.EmitIL(method);
                     }
                     break;
-                case "EEType":
+                case "MethodTable":
```

Comment:
Does this mean that any other custom runtime will break if `bool SupportsRelativePointers` declared in source code?

Reply:
Yep